third_party/pocketfft/BUILD.bazel

```diff
@@ -4,8 +4,21 @@ package(default_visibility = ["//visibility:public"])
 
 cc_library(
     name = "pocketfft",
-    hdrs = ["pocketfft_hdronly.h"],
-    copts = ["-fexceptions"],
```

Comment:
This would need to be reformatted, but I can do that as part of the merge.
(We just apply this tool: https://github.com/bazelbuild/buildtools/blob/master/buildifier/README.md if you're curious.)

Reply:
Thanks, will have a look!